src/app/players/player.component.html

```diff
@@ -0,0 +1,4 @@
+<p>
+  player works!
+  {{player.name}}
```

Comment:
i'm trying to build the functionality of the show pages, but can't get it to even display. any ideas? @merryt @AndrewKF @s-espinosa ?